packages/react-aria-components/src/DateField.tsx

```diff
@@ -61,6 +61,7 @@ function DateField<T extends DateValue>(props: DateFieldProps<T>, ref: Forwarded
   let [labelRef, label] = useSlot();
   let inputRef = useRef<HTMLInputElement>(null);
   let {labelProps, fieldProps, inputProps, descriptionProps, errorMessageProps} = useDateField({...props, label, inputRef}, state, fieldRef);
+  fieldProps = removeDataAttributes(fieldProps);
```

Comment:
I saw Daniel's comment on ComboBox about putting this on the props passed into the hook instead of on the result. Should we do the same for all of the other components too?

Reply:
+1 to this, I think we should follow the same pattern for the other components. Other than that, things look good to me.

Reply:
Just pushed a change to use the same pattern for all components to filter props as they are passed to the hooks.